### ISSUE TITLE
"An illegal reflective access operation has occurred" for com.google.inject.internal.cglib.core.$ReflectUtils$1

### ISSUE THREAD

**Issue by Talkless** [bisq-network/bisq#4144]
### Description
When building v1.3.0 on Sid, I got this warning:
```
> Task :seednode:test
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/home/vincas/.gradle/caches/modules-2/files-2.1/com.google.inject/guice/4.2.2/6dacbe18e5eaa7f6c9c36db33b42e7985e94ce77/guice-4.2.2.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

#### Version

v1.3.0

### Steps to reproduce

`./gradlew build` on Sir.

### Expected behaviour

Should build without scary warnings.

### Actual behaviour

Emits warning.

### Screenshots

N/A.

#### Device or machine

Debian Sid, amd64,
```
$ java -version
openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Debian-2)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Debian-2, mixed mode)
```

#### Additional info

N/A

**Comment by Talkless** [comment]
I still get that messge:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/home/vincas/.gradle/caches/modules-2/files-2.1/com.google.inject/guice/4.2.2/6dacbe18e5eaa7f6c9c36db33b42e7985e94ce77/guice-4.2.2.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by cd2357** [comment]
Related to #4600

As mentioned in https://github.com/bisq-network/bisq/pull/4600#issuecomment-704096224:
> I guess it is fine as we are not using Java 9 modules so far. It seems to be fixed in [google/guice@85e30be](https://github.com/google/guice/commit/85e30beafe55551a649025d3c12e831214057412), but not released yet.

The issue has been tracked by guice for 3 years in https://github.com/google/guice/issues/1133, but the fix got merged in their repo a few months ago in July (https://github.com/google/guice/issues/1133#issuecomment-656906686).

Once they [release a new version](https://github.com/google/guice/releases/), upgrading to that should fix this issue.

**Comment by ripcurlx** [comment]
As [5.0.1](https://github.com/google/guice/releases/tag/5.0.1) got released in February and no hotfix release happend since then I think it is safe to upgrade.
